.github/workflows/benchmark.yml

```
@@ -0,0 +1,111 @@
+          sed -i "s/PHP_COMMIT=/PHP_COMMIT=$YESTERDAY_SHA/g" ./php-version-benchmarks/config/php/master1.ini
+          cp ./php-version-benchmarks/config/php/master.ini.dist ./php-version-benchmarks/config/php/master2.ini
+          cp ./php-version-benchmarks/config/php/master_jit.ini.dist ./php-version-benchmarks/config/php/master2_jit.ini
+
+          cp ./php-version-benchmarks/config/test/1_laravel.ini.dist ./php-version-benchmarks/config/test/1_laravel.ini
+          cp ./php-version-benchmarks/config/test/2_symfony_main.ini.dist ./php-version-benchmarks/config/test/2_symfony_main.ini
+          cp ./php-version-benchmarks/config/test/4_wordpress.ini.dist ./php-version-benchmarks/config/test/4_wordpress.ini
+          cp ./php-version-benchmarks/config/test/5_bench.php.ini.dist ./php-version-benchmarks/config/test/5_bench.php.ini
+          cp ./php-version-benchmarks/config/test/6_micro_bench.php.ini.dist ./php-version-benchmarks/config/test/6_micro_bench.php.ini
+
+          rm -rf ./php-version-benchmarks/docs/results
+      - name: Git setup
+        run: |
+          git config --global user.name "Benchmark"
+          git config --global user.email "benchmark@php.net"
+      - name: Checkout benchmark results
+        uses: actions/checkout@v4
+        with:
+          repository: kocsismate/php-version-benchmark-results
+          ssh-key: ${{ secrets.PHP_VERSION_BENCHMARK_RESULTS_DEPLOY_KEY }}
+          path: 'php-version-benchmarks/docs/results'
+      - name: Run benchmark
+        run: ./php-version-benchmarks/benchmark.sh run aws
+      - name: Store results
+        run: |
+          set -ex
+
+          cd ./php-version-benchmarks/docs/results
+          git pull --autostash
+          if [ -e ".git/MERGE_HEAD" ]; then
+            echo "Merging, can't proceed"
+            exit 1
+          fi
+          git add .
+          if git diff --cached --quiet; then
+            exit 0
```

Should this fail the job? (this seems unexpected)

---

Yes, I guess it's a good idea. I copy-pasted this code from Ilija's benchmark, but didn't notice the 0 exit status.